### ISSUE TITLE
Archived dependency

### ISSUE THREAD

**Issue by sgirardin** [Acosix/alfresco-audit#6]
Hi Axel,

I wanted to use your module and passed it through a dependency security check and the following dependency raised a lot of flags. The project is archived by alfresco and the latest version 20.71 still raises red flags because of its dependency on org.alfresco:alfresco-repository@6.55.
https://github.com/Acosix/alfresco-audit/blob/9e64580b2b0b4ae0131d391ad53eb6000c44dd0b/pom.xml#L108

I tried to find a maintained version but didn't find anything. Alfresco also changed its versioning strategy and it's hard to follow which is the latest. Do you have any idea what could be done?

**Comment by AFaust** [comment]
I am not sure I fully understand. `alfresco-remote-api` in version `20.71` should definitely not depend on `alfresco-repository` in version `6.55`. Specifically in the POM of `alfresco-remote-api` the dependency [is declared to be to the exact same version](https://github.com/Alfresco/alfresco-community-repo/blob/master/remote-api/pom.xml#L14) as its own, so `alfresco-remote-api@20.71` depends on `alfresco-repository@20.71`. I could imagine that you updated the dependency version of `alfresco-remote-api` in the `alfresco-audit` POM and the `alfresco-repository` then did not update accordingly - this is due to the fact that the project is inheriting managed dependency versions from the project parent POM for a very specific Alfresco version. Using managed dependencies unfortunately had become necessary since Alfresco did not provide properly consistent bill of materials and/or correct transitive dependencies for specific releases to build against.
You could instead try to change the parent POM from `de.acosix.alfresco.maven.project.parent-6.0.7` to `de.acosix.alfresco.maven.project.parent-7.0.0` to update the ACS reference bill-of-materials, and see if that fixes your dependency check. I have not created parent project POMs for ACS 7.1, 7.2, or 7.3 as time has not permitted it and they have not been relevant for developing reusable modules yet (as base versions).

**Comment by AFaust** [comment]
Bear in mind: When updating to the 7.0.0 parent POM, the FindBugs plugin must be deactivated as it is incompatible with JDK 11+ (via `<quality.findBugs.skip>true</quality.findBugs.skip>`)

**Comment by sgirardin** [comment]
Hi Axel, 
much thanks for response. 
> Specifically in the POM of `alfresco-remote-api` the dependency [is declared to be to the exact same version](https://github.com/Alfresco/alfresco-community-repo/blob/master/remote-api/pom.xml#L14) as its own, so `alfresco-remote-api@20.71` depends on `alfresco-repository@20.71`. I could imagine that you updated the dependency version of `alfresco-remote-api` in the `alfresco-audit` POM and the `alfresco-repository` then did not update accordingly - this is due to the fact that the project is inheriting managed dependency versions from the project parent POM for a very specific Alfresco version.

Yes it's exactly what I did. I'll try it your way 

> You could instead try to change the parent POM from de.acosix.alfresco.maven.project.parent-6.0.7 to de.acosix.alfresco.maven.project.parent-7.0.0 to update the ACS reference bill-of-materials, and see if that fixes your dependency check. I have not created parent project POMs for ACS 7.1, 7.2, or 7.3 as time has not permitted it and they have not been relevant for developing reusable modules yet (as base versions).

I'll have a look at that.
